app/models/vm_or_template.rb

```diff
@@ -234,6 +234,8 @@ def from_infra_manager?
     'vmsafe.failOpen'     => [:vmsafe_fail_open,     :boolean],
     'vmsafe.immutableVM'  => [:vmsafe_immutable_vm,  :boolean],
     'vmsafe.timeoutMS'    => [:vmsafe_timeout_ms,    :integer],
+    'processor_type'      => [:processor_share_type, :string],
+    'entitled_processors' => [:entitled_processors, :float],
```

Comment:
Minor but can you align the `:float` with the others?  If you look at e.g. https://github.com/ManageIQ/manageiq/pull/21306/files#diff-1cd4edb18c242ba5e1f5795da3f815cde970e5e9672cdd7d96b1a101fe7c465fR236 they are padded to align with the rest